docs/reference/migration/migrate_8_0/packaging.asciidoc

```diff
@@ -0,0 +1,10 @@
+[float]
+[[breaking_80_packaging_changes]]
+=== Packaging changes
+
+[float]
+//tag::notable-breaking-changes[]
+==== Java 11 is required
+
+Java 11 is now required to run Elasticsearch and any of its command line tools.
```

Comment:
do we need to say "11 or greater" to clarify that more recent versions are fine?